helpers/test.py

```diff
@@ -51,7 +51,7 @@ def run_tests(self):
             import pytest
             if not self.pytest_args:
                 self.pytest_args = ['sherpa',]
-            errno = pytest.main(self.pytest_args)
+            errno = pytest.main([self.pytest_args,])
```

Comment:
If the previous condition was fired, then this is going to

    pytest.main([['sherpa',], ])

Is this okay?